emperor/support_files/js/abc-view-controller.js

```diff
@@ -183,5 +183,14 @@ define([
     throw Error('Not implemented');
   };
 
+  /**
+   * Writes the current settings to the active decomposition view(s).
+   * Will be called when the decomposition view is swapped out
+   * for a different view type.
+   */
```

Comment:
- It looks like there's a few unit tests that need to be fixed or moved over. Let me know if you would like help with this.
    - Namely ScenePlotView3D, MultiModel and DecompositionView will need a fair amount of work.